swift-sdk/IterableAppIntegrationInternal.swift

```diff
@@ -205,4 +229,11 @@ struct IterableAppIntegrationInternal {
         return itbl
     }
 
+    private static func legacyDefaultActionFromPayload(userInfo: [AnyHashable : Any]) -> IterableAction? {
+        if let deeplinkUrl = userInfo[ITBConsts.Payload.deeplinkUrl] as? String {
```

Comment:
I don't think ITBConsts.Payload.deeplinkUrl is set correctly, currently it references the constant: ITBL_PAYLOAD_DEEP_LINK_URL.

Shouldn't it be `URL` which is passed down from the backend?

Reply:
Nevermind i see ITBL_PAYLOAD_DEEP_LINK_URL defined in the constants.

Reply:
However, do you need to check case in-sensitivity?

Reply:
Right now we are not checking case insensitivity anywhere we are serializing/deserializing. Do we need to? I think we should be ok as long as this convention is followed by both iOS SDK and backend.

Reply:
The only reason I ask is that the documentation has `URL` as the key being passed down. https://support.iterable.com/hc/en-us/articles/115000315806-Setting-Up-iOS-Push-Notifications

Reply:
Docs are wrong :)

Reply:
Not any more they aren't :)

Reply:
This seems wrong in the doc
```if (userInfouserInfo[deeplinkKey])```

Reply:
Fixed the userInfo in the docs.